salt/proxy/rest_sample.py

```diff
@@ -39,14 +38,15 @@ def __virtual__():
 
 def init(opts):
     log.debug("rest_sample proxy init() called...")
-    DETAILS["initialized"] = True
+    __context__["rest_sample"] = {}
+    __context__["rest_sample"]["initialized"] = True
 
     # Save the REST URL
-    DETAILS["url"] = opts["proxy"]["url"]
+    __context__["rest_sample"]["url"] = opts["proxy"]["url"]
```

Comment:
The whole `__context__["rest_sample"]` dictionary can be constructed in full as suggested on the previous file.